index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>A Product Landing Page</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header id="header">
+        <div id="header-div">
+            <img id="header-img" height="50" width="250"
+                src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+                alt="original trombones logo">
+            <nav id="nav-bar">
+                <ul>
+                    <li><a class="nav-link" href="#features">Features</a></li>
+                    <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
+                    <li><a class="nav-link" href="#pricing">Pricing</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+
+    <div>
```

Comment:
[OPTIONAL] It's very good in requesting this change, however, it will be very nice if you could add emojis to your inline comments. Emojis can enhance communication by adding visual cues to your comments. They can help convey emotions, tone, or intent that may not be immediately apparent from text alone. For example, using a thumbs-up emoji can indicate approval or agreement, while a confused face emoji can indicate that something is unclear. :slightly_smiling_face: